### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by TimNanos**
Improved code readability, effectiveness, performance.

### CODE REVIEW

File: PL_FPDF.pkb [hunk 1]
```diff
@@ -910,15 +904,15 @@ end p_putstream;
...
 	buf_size integer := 2000;
-	buf raw(2000);
+	buf varchar2(2000);
 begin
 	p_out('stream');
 	-- read the blob and put it in small pieces in a varchar
 	while offset < lv_content_length loop
 	  dbms_lob.read(pData,buf_size,offset,buf);
-	  p_out(utl_raw.cast_to_varchar2(buf), false);
+	  p_out(buf, false);
```
Comment (Pilooz) [inline PL_FPDF.pkb]: Not sure it works with dbms_lob.read. does it ?

File: PL_FPDF.pkb [hunk 2]
```diff
@@ -1960,19 +1944,6 @@ begin
 	return FontFamily;
 end GetCurrentFontFamily;
 
-
```
Comment (Pilooz) [inline PL_FPDF.pkb]: Why did you erase this procedure ?

**Comment by Pilooz** [comment]
Hi, 
It seems to be great. Just want to know the reason why the procedure "ln" disappear ? Don't you think it will break backward compatibility ?

File: PL_FPDF.pkb (hunk 1 above)
Comment (TimNanos) [inline PL_FPDF.pkb]: It does, DBMS_LOB.READ can also accept VARCHAR2 as the output buffer parameter:
https://docs.oracle.com/cd/B28359_01/appdev.111/b28419/d_lob.htm#i999170

File: PL_FPDF.pkb (hunk 2 above)
Comment (TimNanos) [inline PL_FPDF.pkb]: Ooops!

**Comment by TimNanos** [comment]
Hi!
Sorry, my bad. That happened accidentally.
I added it again, so it should be fine now.
Thanks!

**Comment by Pilooz** [comment]
Ok, Thanks for the job !

**Comment by TimNanos** [comment]
Thank you!

**Pull request merged**